src/assets/packs/species-templates.db

```diff
@@ -0,0 +1,4 @@
i5tJtAE\")","get-item-like-this-macro":"await game.system.api.rqid.fromRqid(\"rune-movement-power\",\"en\")"},"effects":[],"folder":null,"sort":0,"permission":{"default":0,"7aFAXAK1KTRDDeBB":3},"flags":{"core":{"sourceId":"Compendium.starter-set-en-rqg.runes.MyGTXyDtL7SNaiun"}}},{"_id":"TaOrFN646OemkARh","name":"Man (form)","type":"rune","img":"systems/rqg/assets/images/runes/man.svg","data":{"rqid":"i.rune-man-form","rqidPriority":0,"rqidLang":"en","rune":"Man","chance":50,"canGetExperience":true,"hasExperience":false,"runeType":"form","opposingRune":"Beast (form)","minorRunes":[],"isMastered":false,"journalId":"Y5rHARO63TIb6K7C","journalPack":"starter-set-en-rqg.rune-descriptions","journalName":""},"effects":[],"folder":null,"sort":0,"permission":{"default":0,"7aFAXAK1KTRDDeBB":3},"flags":{"core":{"sourceId":"Compendium.starter-set-en-rqg.runes.Qj8RbQcOzvp6YnOT"}}},{"_id":"PEcXEAP8NuImW8po","name":"Beast (form)","type":"rune","img":"systems/rqg/assets/images/runes/beast.svg","data":{"rqid":"i.rune-beast-form","rqidPriority":0,"rqidLang":"en","rune":"Beast","chance":50,"canGetExperience":true,"hasExperience":false,"runeType":"form","opposingRune":"Man (form)","minorRunes":[],"isMastered":false,"journalId":"twWC6a757e2jSQpB","journalPack":"starter-set-en-rqg.rune-descriptions","journalName":""},"effects":[],"folder":null,"sort":0,"permission":{"default":0,"7aFAXAK1KTRDDeBB":3},"flags":{"core":{"sourceId":"Compendium.starter-set-en-rqg.runes.gWDzwBRC6ukVbhY5"}}},{"_id":"g9bZQQCAgOpirFIs","name":"Honor","type":"passion","img":"systems/rqg/assets/images/passion/honor.svg","data":{"rqid":"i.passion-honor","rqidPriority":0,"rqidLang":"en","passion":"Honor","subject":"","description":null,"gmNotes":"<p>asdf</p>","chance":60,"canGetExperience":true,"hasExperience":false,"rqidpriority":null,"rqidlang":"en","rqidlocale":"","item-foundryid":"Reb4mkS2eowI0jKk","item-foundryuuid":"Compendium.world.passions.Reb4mkS2eowI0jKk","get-item-macro":"await fromUuid(\"Compendium.world.passions.Reb4mkS2eowI0jKk\")","get-item-like-this-macro":"await game.system.api.fromRqid(\"passion-honor\",\"en\")"},"effects":[],"folder":null,"sort":0,"permission":{"default":0,"7aFAXAK1KTRDDeBB":3},"flags":{"core":{"sourceId":"Item.Reb4mkS2eowI0jKk"}}},{"_id":"PGRdT4Fsw11D9NjF","name":"Odalya (condition)","type":"rune","img":"systems/rqg/assets/images/runes/odayla.svg","data":{"rqid":"rune-odalya-condition","rqidPriority":"3000","rqidLang":"en","rune":"Odalya","chance":0,"canGetExperience":true,"hasExperience":false,"runeType":"condition","opposingRune":null,"minorRunes":[],"isMastered":false,"journalId":"","journalPack":"","journalName":""},"effects":[],"folder":null,"sort":0,"permission":{"default":0,"7aFAXAK1KTRDDeBB":3},"flags":{"core":{"sourceId":"Compendium.starter-set-en-rqg.runes.vcXrFRyhdz5nqlpV"}}}],"effects":[],"folder":null,"sort":0,"permission":{"default":0,"7aFAXAK1KTRDDeBB":3},"flags":{"core":{"sourceId":"Actor.4vG40EH4XPT58UWu"},"rqg":{"":"","is-actor-template":true}}}
```

Comment:
> The db is saved in a non compressed state, so there are 4 lines when there should only be 2. Easy fix is to shut down foundry before copying the db file to the repo, since foundry will compact the db file while it shuts down.

I've been doing that but it doesn't always work.  I do think we need to get the yaml setup from the starter set working for the core so we can more easily maintain the packs we have there.

Reply:
In the starter set I'm using a gulp build pipeline, but here I'm just using a plain build without any extra. I'll see what can be done for that though. I'm copying the other system packs from the starter set build and cleaning them of things that shouldn't be there manually, so it would be good to get a better build flow.